board/sharp/common/dma.c

```diff
@@ -4,4 +4,6 @@
 
```

Comment:
(must) Implicit declaration 出てて include で修正できました。

```suggestion
#include <asm/arch/sys_proto.h>

```